### DISCUSSION TITLE
How to save settings

### DISCUSSION THREAD

**Discussion by hmoscovitz** [AlexandreRouma/SDRPlusPlus#427] [Q&A]
I'm using sdr++ on a Windows 10 maching. Thanks for providing this for us. 

I've looked and looked but can't find it.  How do you save one's settings?

**Comment by AlexandreRouma** [comment]
All settings are automatically saved to files in the main directory.
if you want to back up and restore that configuration, you have to cppy all files ending in `config.json`

**Comment by adevivo** [comment]
on Mac OS it seems that the Scanner module settings are not saved.

**Reply by AlexandreRouma** [reply]
They aren't saved on **any** OS. If you had read the [manual](https://www.sdrpp.org/manual.pdf) or the [readme](https://github.com/AlexandreRouma/SDRPlusPlus#misc) you'd have seen that it's in beta and doesn't save the settings.

**Comment by gweep** [comment]
There is a need for savable and retrievable settings. My rtl sdr unit has an up converter that I switch out for VHF.  Hence the need for a second set of settings for a single SDR stick. 

In my case a check box to enable/disable the frequency offset would do the job.
